package.json

```diff
@@ -10,6 +10,13 @@
     "globals"
   ],
   "main": "index.js",
+  "typings": "index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./index.d.ts",
+      "default": "./index.js"
+    }
+  },
```

Comment:
Could you explain these fields? Any documentation?

Reply:
Here's the documentation for exports: https://nodejs.org/api/packages.html#package-entry-points.
And here's the documentation for explicit `"types"` https://www.typescriptlang.org/docs/handbook/modules/reference.html#packagejson-exports